### PR TITLE
log date, tz, millis

### PR DESCRIPTION
**Pull request by phritz**

### CODE REVIEW

File: repm/repm.go [hunk 1]
```diff
@@ -47,9 +47,9 @@ type Logger interface {
 // is created. Logger receives logging output from Replicache.
 func Init(storageDir, tempDir string, logger Logger) {
 	if logger == nil {
-		zlog.Logger = zlog.Output(zl.ConsoleWriter{Out: os.Stdout})
+		zlog.Logger = zlog.Output(zl.ConsoleWriter{Out: os.Stdout, TimeFormat: "02 Jan 06 15:04:05.000 -0700"})
```
Comment (aboodman) [inline repm/repm.go:50]: wedding day?

Comment (aboodman) [reply repm/repm.go:50]: Kinda high resolution for that though.

Comment (arv) [reply repm/repm.go:50]: [fyi] That date is part of the API. https://golang.org/pkg/time/#Time.Format

**Comment by aboodman** [comment]
Hm, wonder what that specific date is though.

**Comment by arv** [comment]
I think it is picked because: 1 (jan), 2 (date of month), 3 for hours (in our case 15), 4 (minutes), 5 (seconds), 6 (milliseconds), 7 (timezone)

**Comment by aboodman** [comment]
aw nuts. that's a lot less romantic than one might hope.

File: repm/repm.go (hunk 1 above)
Comment (phritz) [reply repm/repm.go:50]: the thing that makes this comment funny for me is that @aboodman i think we have the same wedding day, which is why I think you couldn't come to mine...

Comment (aboodman) [reply repm/repm.go:50]: lol, whoopsie

**Pull request merged**